src/ViewRendererBundle/Bridge/Chameleon/Migration/Script/update-1539324183.inc.php

```diff
@@ -0,0 +1,38 @@
+<h1>update - Build #1539324183</h1>
+<h2>Date: 2018-10-12</h2>
+<div class="changelog">
+    - #117: Only use necessary additional cache settings
+</div>
+<?php
+
+$connection = TCMSLogChange::getDatabaseConnection();
+$oldValue = trim($connection->fetchColumn('SELECT `additional_files_to_delete_from_cache` FROM `cms_config`'));
+$newValue = getSnippetChainWithElementRemoved($oldValue, 'chameleon/outbox/static/less/cached');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_config', 'de')
+  ->setFields(array(
+      'additional_files_to_delete_from_cache' => $newValue,
+  ))
+  ->setWhereEquals(array(
+      'id' => '1',
+  ))
+;
+TCMSLogChange::update(__LINE__, $data);
+
+// Helpers copied from SnippetChainModifier:
+function optimizeSnippetChainString(string $snippetChain): string
```

Comment:
Please don't define methods, classes or constants in migration files. When running the migrations from console there might be conflicts if multiple scripts define the same identifiers (even years later - remember these functions get defined on the global namespace).

Also the terms "snippet" and "snippet chain" are misleading here as they are normally used for view snippets.

Reply:
ok